pkg/ccl/testccl/sqlstatsccl/sqlstatsccl_test: TestSQLStatsRegions failed

pkg/ccl/testccl/sqlstatsccl/sqlstatsccl_test.TestSQLStatsRegions [failed](https://tanzanite.cluster.engflow.com/invocations/default/5f767eae-57b7-4be9-9575-418e8ebd2dfe?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3Fsc3RhdHNjY2w6c3Fsc3RhdHNjY2xfdGVzdA==) on release-25.1 @ [0a14af43960ce2fcb79a2624d3b94fe2ce3662e2](https://github.com/cockroachdb/cockroach/commits/0a14af43960ce2fcb79a2624d3b94fe2ce3662e2):


```
=== RUN   TestSQLStatsRegions
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestSQLStatsRegions3383140958
    test_log_scope.go:76: use -show-logs to present logs inline
=== RUN   TestSQLStatsRegions/secondary_tenant
=== RUN   TestSQLStatsRegions/system_tenant
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/obs-prs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSQLStatsRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-46726

pkg/ccl/testccl/sqlstatsccl/sqlstatsccl_test.TestSQLStatsRegions [failed](https://tanzanite.cluster.engflow.com/invocations/default/34d26803-28e5-4b40-a454-7c89b2b21441?testReportRun=18&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3Fsc3RhdHNjY2w6c3Fsc3RhdHNjY2xfdGVzdA==) on release-25.1 @ [98ef65420f236cb16e7d88767ce3e38eab1e9b54](https://github.com/cockroachdb/cockroach/commits/98ef65420f236cb16e7d88767ce3e38eab1e9b54):

Fatal error:

```
panic: test timed out after 5m0s
running tests:
	TestSQLStatsRegions (5m0s)

```
Stack: 

```
goroutine 221951 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestSQLStatsRegions
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestSQLStatsRegions2168229011
    test_log_scope.go:76: use -show-logs to present logs inline
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=18</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSQLStatsRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/testccl/sqlstatsccl/sql_stats_test.go
+++ b/pkg/ccl/testccl/sqlstatsccl/sql_stats_test.go
@@ -84,26 +84,6 @@ func TestSQLStatsRegions(t *testing.T) {
 	}()
 
 	tdb := sqlutils.MakeSQLRunner(host.ServerConn(0))
-
-	// Max out rate limit for replica rebalances.
-	tdb.Exec(t, `set cluster setting kv.snapshot_rebalance.max_rate = '1g'`)
-
-	// Shorten the closed timestamp target duration so that span configs
-	// propagate more rapidly.
-	tdb.Exec(t, `SET CLUSTER SETTING kv.closed_timestamp.target_duration = '200ms'`)
-	tdb.Exec(t, "SET CLUSTER SETTING kv.allocator.load_based_rebalancing = off")
-
-	// Lengthen the lead time for the global tables to prevent overload from
-	// resulting in delays in propagating closed timestamps and, ultimately
-	// forcing requests from being redirected to the leaseholder. Without this
-	// change, the test sometimes is flakey because the latency budget allocated
-	// to closed timestamp propagation proves to be insufficient. This value is
-	// very cautious, and makes this already slow test even slower.
-	tdb.Exec(t, "SET CLUSTER SETTING kv.closed_timestamp.side_transport_interval = '10ms'")
-	tdb.Exec(t, `SET CLUSTER SETTING kv.closed_timestamp.lead_for_global_reads_override = '500ms'`)
-	tdb.Exec(t, "SET CLUSTER SETTING kv.rangefeed.closed_timestamp_refresh_interval = '10ms'")
-	tdb.Exec(t, `ALTER TENANT ALL SET CLUSTER SETTING spanconfig.reconciliation_job.checkpoint_interval = '500ms'`)
-
 	tdb.Exec(t, `ALTER RANGE meta configure zone using constraints = '{"+region=gcp-us-west1": 1, "+region=gcp-us-central1": 1, "+region=gcp-us-east1": 1}';`)
 
 	// Create secondary tenants

